packages/next/server/api-utils.ts

```diff
@@ -321,19 +324,6 @@ export function sendData(
   res.end(stringifiedBody)
 }
 
-/**
- * Send `JSON` object
- * @param res response object
- * @param jsonBody of data
- */
-export function sendJson(res: NextApiResponse, jsonBody: any): void {
-  // Set header to application/json
-  res.setHeader('Content-Type', 'application/json; charset=utf-8')
-
-  // Use send to handle request
-  res.send(jsonBody)
```

Comment:
Instead of changing the behavior of `res.send` do we want to just do `res.send(JSON.stringify(jsonBody))` here which should match the middleware `Response` helper here?

https://github.com/vercel/next.js/blob/26c40f31860a9e35782bcf8e5b995c5f44731eaf/packages/next/server/web/spec-extension/response.ts#L70-L74